sql/core/src/test/resources/sql-tests/results/order-by-ordinal.sql.out

```diff
@@ -14,19 +14,6 @@ struct<>
 
 
 
--- !query
-select * from data order by 1 desc
```

Comment:
why do we remove this test?

Reply:
Reverted it and change it to `select * from data order by 1 desc, 2;`. otherwise sometimes it fails:
```
[info] - order-by-ordinal.sql *** FAILED *** (1 second, 39 milliseconds)
[info]   order-by-ordinal.sql
[info]   Expected "3    2
[info]   3      1
[info]   2      [2
[info]   2      1]
[info]   1      1
[info]   1      2", but got "3  2
[info]   3      1
[info]   2      [1
[info]   2      2]
[info]   1      1
[info]   1      2" Result did not match for query #1
[info]   select * from data order by 1 desc (SQLQueryTestSuite.scala:478)

```